libunwind/src/DwarfInstructions.hpp

```diff
@@ -20,7 +20,9 @@
 #include "Registers.hpp"
 #include "DwarfParser.hpp"
 #include "config.h"
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_CHERI_C18N_SUPPORT)
```

Comment:
The former should really be implied by the latter

Reply:
Though at this point the implementation is probably minimal and non-invasive enough that, unless you #ifdef all the assembly to avoid the handful of extra instructions, there's not much point having the option at all.